? policy/policy.20
? policy/policy.conf
? policy/tmp
? policy/file_contexts/file_contexts
? policy/file_contexts/homedir_template
? policy/file_contexts/misc
Index: policycoreutils/semodule_expand/semodule_expand.c
===================================================================
RCS file: /cvsroot/selinux/nsa/selinux-usr/policycoreutils/semodule_expand/semodule_expand.c,v
retrieving revision 1.9
diff -u -r1.9 semodule_expand.c
--- policycoreutils/semodule_expand/semodule_expand.c	20 Oct 2005 16:37:23 -0000	1.9
+++ policycoreutils/semodule_expand/semodule_expand.c	5 Feb 2006 20:28:13 -0000
@@ -36,8 +36,8 @@
 
 int main(int argc, char **argv)
 {
-	char ch, *basename, *outname;
-	int ret, show_version = 0, verbose = 0;
+	char *basename, *outname;
+	int ch, ret, show_version = 0, verbose = 0;
 	struct sepol_policy_file *pf;
 	sepol_module_package_t *base;
 	sepol_policydb_t *out;
Index: policycoreutils/semodule_link/semodule_link.c
===================================================================
RCS file: /cvsroot/selinux/nsa/selinux-usr/policycoreutils/semodule_link/semodule_link.c,v
retrieving revision 1.7
diff -u -r1.7 semodule_link.c
--- policycoreutils/semodule_link/semodule_link.c	20 Oct 2005 16:37:23 -0000	1.7
+++ policycoreutils/semodule_link/semodule_link.c	5 Feb 2006 20:28:13 -0000
@@ -73,8 +73,8 @@
 
 int main(int argc, char **argv)
 {
-	int i, show_version = 0, verbose = 0, num_mods;
-	char ch, *basename, *outname = NULL;
+	int ch, i, show_version = 0, verbose = 0, num_mods;
+	char *basename, *outname = NULL;
 	sepol_module_package_t *base, **mods;
 	FILE *outfile;
 	struct sepol_policy_file *pf;
Index: policycoreutils/sestatus/sestatus.c
===================================================================
RCS file: /cvsroot/selinux/nsa/selinux-usr/policycoreutils/sestatus/sestatus.c,v
retrieving revision 1.9
diff -u -r1.9 sestatus.c
--- policycoreutils/sestatus/sestatus.c	29 Nov 2005 15:55:02 -0000	1.9
+++ policycoreutils/sestatus/sestatus.c	5 Feb 2006 20:28:13 -0000
@@ -160,9 +160,8 @@
 
 int main(int argc, char **argv) {
 	/* these vars are reused several times */
-	int rc, i,c;
+	int rc,opt, i,c;
 	char *context;
-	char opt;
 
 	/* files that need context checks */
 	char *fc[MAX_CHECK];
